packages/angular_devkit/core/src/json/schema/visitor.ts

```diff
@@ -90,6 +110,18 @@ function _visitJsonRecursive(
               value[key],
               visitor,
               joinJsonPointer(ptr, key),
+              (typeof schema == 'object'
```

Comment:
This really looks like it should be in a helper function with some explanation of what the purpose is.